test/acceptance/features/steps/steps.py

```diff
@@ -436,3 +437,34 @@ def update_binding_secret(context):
     output = openshift.apply(yaml.dump(secret_yaml), context.namespace.name)
     result = re.search(rf'.*{context.sb_secret}.*(created|unchanged|configured)', output)
     assert result is not None, f"Unable to apply YAML for binding secret '{context.sb_secret}': {output}"
+
+
+@then(u'cluster role "{cluster_role_name}" is available in the cluster')
+def check_cluster_role_exists(context, cluster_role_name):
+    openshift = Openshift()
+    roles_present, exit_code = openshift.is_resource_in("clusterroles")
+    if exit_code == 0:
+        assert cluster_role_name in roles_present, f"Could not found the cluster role : '{cluster_role_name}' in the cluster"
+    else:
+        assert False, "Cluster role does not exist in the cluster"
+
+
+@then(u'operator service account is bound to "{cluster_role_name}" cluster role')
+def check_service_account_bound(context, cluster_role_name):
+    openshift = Openshift()
+    sbr = Servicebindingoperator()
+    operator_namespace = sbr.check_deployment()
+    (rolebinding, exitcode) = openshift.get_json_resource("clusterrolebinding", "service-binding-operator-controller-rolebinding", operator_namespace)
```

Comment:
see my earlier comment - the code can be simplified if assert exit code in `get_json_resource`